### PR TITLE
Update news in preparation for a new release

### PR DESCRIPTION
**Pull request by vzh**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -2,33 +2,111 @@ Recent Changes in Lepton EDA
 ============================
 
 This file documents important user-visible changes in Lepton EDA.  For
-more information, please consult the `ChangeLog` file.
```
Comment (graahnul-grom) [inline NEWS.md]: Shouldn't we delete top-level `ChangeLog` file (or at least bring it up to date)?

File: NEWS.md [hunk 2]
```diff
@@ -59,7 +137,459 @@ Notable changes in Lepton EDA 1.9.5
...
+- The configure script now supports a `--disable-deprecated` switch.
+  When specified, various deprecated features and behaviour are
+  disabled, which may break backward-compatibility with some
+  configurations.
+
 ----------------------------------------------------------------
 
-Please see [docs/NEWS-1.9.4.md](docs/NEWS-1.9.4.md) for info on
-pre-1.9.5 changes.
+Please see [NEWS-1.9.2.txt](NEWS-1.9.2.txt) for info on
```
Comment (graahnul-grom) [inline NEWS.md]: Do you plan to move `NEWS-1.9.2.txt`? Otherwise, the hyperlink is broken.

File: NEWS.md (hunk 1 above)
Comment (vzh) [reply NEWS.md]: OK, I've deleted it.

File: NEWS.md (hunk 2 above)
Comment (vzh) [reply NEWS.md]: This seems to be fixed now, thanks!

**Pull request merged**